css/index.css

```diff
@@ -11,4 +11,98 @@ body{
 } 
 
 /* Set a style for the rest of the elements besides the header elements on the index.html page */
-                                 
+#parent {
+    display: flex;
+align-items: center;
+justify-content: center;
+margin-top: 12%;
+}
+input.postButton {
+    width: 100px;
+    padding: 8px;
+    cursor: pointer;
+    font-weight: bold;
+    font-size: 90%;
+    background: #ecd540;
+    color: black;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+
+
+
+
+form {
```

Comment:
Defined in _header.css_, can be removed.

Reply:
removed from index.css